R/agg.R

```diff
@@ -23,10 +27,10 @@ preprocess <- function(x, q = 0) {
     if (all(x == 0)) {
       # Replace them with 10^-12
       return(rep(10^-12, length(x)))
-    } else if (all(x == 100)) {
-      return(rep(100 - 10^-12, length(x)))
+    } else if (all(x == 1)) {
+      return(rep(1 - 10^-14, length(x)))
```

Comment:
is `10^-14` machine epsilon? Does R have a constant that represents this that you can use instead of hard coding it? again forgive ignorant comments!

Reply:
Oooooo ok I actually don't remember why I picked this value for epsilon. It may well be sort of P8-specific. I'll think about this.

Reply:
@zjac0bs has also asked about this before! Time that I did something about it!

Reply:
Sounds good. It could also become a constant in the code somewhere, especially if it's used elsewhere

Reply:
I'm actually going to wait to change this because it might change P8 results in tiny ways. I want to preserve this version for P8 to depend on. But I'll open an issue for this, to change later.